index.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+require('dotenv').config();
+
+const server = require('./src/server.js')
+const mongoose = require('mongoose');
+const PORT = process.env.PORT;
+
+mongoose.connect(process.env.MONGODB_URI, { 
+    useNewUrlParser: true, 
+    useUnifiedTopology: true,
+    useFindAndModify: false,
+    useCreateIndex: true
+ })
+    .then(() => {
+        server.start();
```

Comment:
Remember you need to tell the server what port to start on, right now it's running on undefined